### ISSUE TITLE
Rubocop does not properly indent method body

### ISSUE THREAD

**Issue by BackOrder** [bbatsov/rubocop#4219]
**`sample.rb`**:
```ruby
class RubocopSampleTest
def m(args)
puts "properly indented"
puts "Why am I not indented?"
puts "how about me?"
end
end
```

--------

## Expected behavior

```ruby
class RubocopSampleTest
  def m(_args)
    puts 'properly indented'
    puts 'Why am I not indented?'
    puts 'how about me?'
  end
end
```

## Actual behavior

```ruby
class RubocopSampleTest
  def m(_args)
    puts 'properly indented'
  puts 'Why am I not indented?'
  puts 'how about me?'
  end
end
```

## Steps to reproduce the problem

Ran with default options (no `.rubocop.yml`) and using command line `rubocop --only-guide-cops -a -D sample.rb`.

```
$ rubocop -V
0.48.0 (using Parser 2.4.0.0, running on ruby 2.2.6 i386-mingw32) **Windows/MSYS2**
0.47.1 (using Parser 2.4.0.0, running on ruby 2.1.5 i386-mingw32) **Windows/cmd**
```

**Comment by pocke** [comment]
The indentation can be corrected by `Style/IndentationConsistency` cop. However, the cop doesn't have a link to the style guide. So, with `--only-guide-cops` option, the indentation is not corrected. 

I think we can fix the problem by adding a link to the style guide for the cop. https://github.com/bbatsov/ruby-style-guide#spaces-indentation

**Comment by bbatsov** [comment]
Definitely.

**Comment by Drenmi** [comment]
I'm not a huge fan on relying on people having a certain *other* cop enabled to clean up a messy auto-correct. I've been making the opposite assumption for the cops I've been working on. 🤔

**Comment by BackOrder** [comment]
Thanks. You guys are fast. The fix does work.
